pkg/image/api/validation/validation.go

```diff
@@ -159,18 +170,33 @@ func ValidateImageStreamMapping(mapping *api.ImageStreamMapping) field.ErrorList
 
 // ValidateImageStreamTag is essentially a no-op.  We don't allow direct creation of istags
 func ValidateImageStreamTag(ist *api.ImageStreamTag) field.ErrorList {
-	return validation.ValidateObjectMeta(&ist.ObjectMeta, true, oapi.MinimalNameRequirements, field.NewPath("metadata"))
+	result := validation.ValidateObjectMeta(&ist.ObjectMeta, true, oapi.MinimalNameRequirements, field.NewPath("metadata"))
+	if ist.Tag != nil {
+		result = append(result, ValidateImageStreamTagReference(*ist.Tag, field.NewPath("tag"))...)
+		if ist.Tag.Annotations != nil && !kapi.Semantic.DeepEqual(ist.Tag.Annotations, ist.ObjectMeta.Annotations) {
+			result = append(result, field.Invalid(field.NewPath("tag", "annotations"), "<map>", "tag annotations must not be provided or must be equal to the object meta annotations"))
```

Comment:
This is a really unusual restriction.  Wouldn't this mean that a future `oc annotate` call would forever fail on these objects?